src/dialogs/notifications/notification-drawer.js

```diff
@@ -69,6 +75,13 @@ export class HuiNotificationDrawer extends EventsMixin(
               </div>
             </template>
           </dom-repeat>
+            <template is="dom-if" if="[[_moreThanOne(notifications)]]">
+              <div class="notification-actions">
+                <ha-progress-button raised="" on-click="_dismissAll">
```

Comment:
does this need to be a progress button? The action is immediate not async

Reply:
It might not need to. I'll see if I can find a more appropriate/simpler button.

Reply:
Replaced it with the standard `mwc-button` (and fixed the indentation).